### PR TITLE
73 show game description in card

### PR DESCRIPTION
**Pull request by valselviz**
Some of the styles for game cards where broken, this MR fixes that.

Also, this MR displays the game description when the single column mode is active.

image

### CODE REVIEW

**Comment by elios11** [comment]
I think the previous ones weren't pixel perfect, did you try this ones with the pixel perfect chrome extension? So that way they are finished and we don't have to worry much more about them.
It's looking pretty good to me, though.

**Review by elios11 — APPROVED** [review]

File: src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.module.css [hunk 1]
```diff
@@ -18,10 +18,9 @@
         font-weight: 700;
         line-height: 2.8rem;
         overflow: hidden;
-        white-space: wrap;
```
Comment (elios11) [inline src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.module.css:21]: This property doesn't exist, I changed it to the correct name in my previous PR.

**Review by elios11 — CHANGES_REQUESTED** [review]

File: src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.module.css [hunk 2]
```diff
@@ -18,10 +18,9 @@
...
+        white-space: rap;
```
Comment (elios11) [inline src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.module.css]: This property doesn't exist, I changed it to the correct name in my previous PR but it's here again for some reason.

**Review by elios11 — CHANGES_REQUESTED** [review]

File: src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.module.css (hunk 2 above)
Comment (valselviz) [reply src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.module.css]: Done!

**Review by elios11 — APPROVED** [review]

**Review by ariadnaflores — APPROVED** [review]

**Pull request merged**